design/README.md

```diff
@@ -1,17 +1,18 @@
-## TAG Security Logos
+## Security TAG Logos
 
-*Note: GitHub Flavored Markdown used in the Readme doesn't support background colors. The white logos below are displayed on the light grey of tables.*
+*Note: GitHub Flavored Markdown used in the Readme doesn't support background colors. The white logos below are displayed on the medium grey (#474756) in its row.*
```

Comment:
Actually, it's not picking up that style.  It seems to be using a lighter grey
![image](https://user-images.githubusercontent.com/41906/128273200-80a052e7-a85e-4a49-877b-2b07d5e9a086.png)

when I look at using the browser inspector, I see...
![image](https://user-images.githubusercontent.com/41906/128273014-f7cac846-e84e-483f-a2b7-53b397024611.png)

It is using the value `#f6f8fa`
![image](https://user-images.githubusercontent.com/41906/128273056-f5222067-bfae-42dd-b3b2-973d4849e46d.png)